src/scripts.js

```diff
@@ -1 +1,28 @@
-console.log("Hello World");
\ No newline at end of file
+const userId = Math.floor(Math.random() * Math.floor(userData.length)) + 1
+const userCard = document.querySelector('.user-card');
+const userGreeting = document.querySelector('.user-greeting');
+
+window.onload = function() {
```

Comment:
maybe we can name this function something to describe that it handling the load lol